trade-finance-manager-ui/templates/case/facility/facility.njk

```diff
@@ -6,6 +6,10 @@
 {% import '../amendments/_macros/amendment-in-progress-bar.njk' as amendmentInProgressBar %}
 {% from "govuk/components/tabs/macro.njk" import govukTabs %}
 
+{% block pageTitle -%}
+  Deal {{ deal.details.ukefDealId }} Facility {{ facility.ukefFacilityId }}
```

Comment:
Same as above?

Reply:
Yes, it works for GEF, other deals, and all Facilities.